regression/scripts/aerodynamics/aerodynamics.py

```diff
@@ -80,6 +80,74 @@ def main():
     T = T[:,None]
     pressure = pressure[:,None]
     
+    Mc = np.array([[0.9  ],
```

Comment:
Where did these come from?

Reply:
This is due to the differences between the random seed in Python 2 and 3. Rather than use a randomizer we hardcoded the randomized values in the regression.